custom_types.yaml

```diff
@@ -580,6 +595,10 @@ node_types:
         description: galaxy version installed
         default: master
         required: false
```

Comment:
Hi @mtangaro , I think that you miss a flavor field here.

Reply:
Hi @micafer, no because we have the derived type tosca.nodes.indigo.GalaxyWnToolDeps with the flavor.
Once galaxy will have fixed (at least hopefully) every conda package, we will not need the second tosca type with the flavor.